atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -0,0 +1,147 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+
+get the image manifest lists from the worker builders. If possible, group them together
+and return them. if not, return the x86_64/amd64 one after tagging it.
```

Comment:
Could use clarification to avoid putting 'manifest' and 'lists' together when not talking about 'manifest list' objects.
```
Get the image manifest digests from the worker builds and group them together into a manifest list.
When told not to group them (group=False), just tag the amd64 image manifest instead.
```